packages/edge-preview-authenticated-proxy/tests/index.test.ts

```diff
@@ -145,9 +155,7 @@ describe("Preview Worker", () => {
 		expect(
 			json.headers.find(([h]: [string]) => h === "cf-workers-preview-token")[1]
 		).toBe(token);
-		expect(json.url).toMatchInlineSnapshot(
-			'"http://preview.devprod.cloudflare.dev/"'
-		);
+		expect(json.url).toMatchInlineSnapshot('"http://127.0.0.1:6756/"');
```

Comment:
How come this has changed? This seems like a behaviour difference that might be important?

Reply:
This fixes a bug in the tests (the remote worker was reading from the wrong `wrangler.toml` file)